flash/core/data/io/output_transform.py

```diff
@@ -85,6 +85,9 @@ def _extract_metadata(batch: Any) -> Tuple[Any, Optional[Any]]:
         return batch, metadata
 
     def forward(self, batch: Sequence[Any]):
+        if batch is None:
```

Comment:
This looks odd, when can `batch` be `None`?

Reply:
We have several tests where the `predict_step` doesn't return anything. As the predict_step is being wrapped with the new API, this would fail. I think it is safer to return the batch is None.